packages/api/src/controllers/session.ts

```diff
@@ -78,6 +79,9 @@ app.get("/", authorizer({}), async (req, res, next) => {
   if (parentId) {
     query.push(sql`session.data->>'parentId' = ${parentId}`);
   }
+  query.push(
+    sql`coalesce(session.data->>'projectId', '') = ${req.project?.id || ""}`
+  );
```

Comment:
We probably don't want to include this when the caller is an admin and has provided the `all` qs (L73), or when they've provided an `userId` qs (some weird condition chain here, might wanna rewrite)